### PR TITLE
GM: Chevy Equinox 2019-22

### PR DESCRIPTION
**Pull request by JMPZ11**
Note: Only applies to vehicles with camera-based ACC. The RPO configuration is called "Adaptive Cruise Control - Camera", "KSG without UGN"

**Checklist**
- [x] added entry to CarInfo in selfdrive/car/*/values.py and ran `selfdrive/car/docs.py` to generate new docs
- [ ] test route added to [routes.py](https://github.com/commaai/openpilot/blob/master/selfdrive/car/tests/routes.py)
- [ ] route with openpilot:
- [ ] route with stock system:

### CODE REVIEW

**Comment by sshane** [comment]
Can your tester drive on this branch and provide a test route?

**Comment by sshane** [comment]
@JasonJShuler can you re-open when you get a test route on this branch?

**Comment by adeebshihadeh** [comment]
@sshane let's leave car port PRs open. We can even merge this as dashcamOnly if you want to make your life easier @JasonJShuler and prevent merge conflicts until we get a test route

File: selfdrive/car/gm/values.py [hunk 1]
```diff
@@ -84,6 +85,7 @@ class GMCarInfo(CarInfo):
   CAR.BUICK_REGAL: GMCarInfo("Buick Regal Essence 2018"),
   CAR.ESCALADE_ESV: GMCarInfo("Cadillac Escalade ESV 2016", "Adaptive Cruise Control (ACC) & LKAS"),
   CAR.BOLT_EUV: GMCarInfo("Chevrolet Bolt EUV 2022-23", "Premier/Premier Redline Trim", footnotes=[], harness=Harness.gm),
+  CAR.EQUINOX: GMCarInfo("Chevrolet Equinox 2018-22", "Premier/LT/RS Trim w Adaptive Cruise Control & LKAS", footnotes=[], harness=Harness.gm),
```
Comment (sshane) [inline selfdrive/car/gm/values.py:88]:
```suggestion
  CAR.EQUINOX: GMCarInfo("Chevrolet Equinox 2018-22", "Advanced Safety Package", footnotes=[], harness=Harness.gm),
```

Comment (sshane) [reply selfdrive/car/gm/values.py:88]: Also, I don't see anything about ACC on the 2018 Equinox

**Comment by sshane** [comment]
Merged as it prevents any more merge conflicts, @JasonJShuler just need a test route now!

**Pull request merged**